dashboard/lib/api/v1/school_autocomplete.rb

```diff
@@ -19,7 +19,10 @@ def get_matches(query, limit)
     if search_by_zip?(query)
       schools = schools.where("zip LIKE ?", "#{query[0, 5]}%")
     else
-      schools = schools.where("MATCH(name,city) AGAINST(? IN BOOLEAN MODE)", to_search_string(query))
+      query = to_search_string(query)
+      schools = schools.
+        where("MATCH(name,city) AGAINST(? IN BOOLEAN MODE)", query).
+        order("MATCH(name,city) AGAINST('#{query}' IN BOOLEAN MODE) DESC, state, city")
```

Comment:
nit: rather than reassigning `query`, use a different name since it's now a completely different format. Maybe `search_string = to_search_string(query)` ?

Reply:
Updated.